src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+                    Log.Error(LogTag, "GetAudioType Failed with error " + error);
+                    return "";
+                }
+
+                return type;
+            }
+            set
+            {
+                ErrorCode error = VcMgrSetAudioType(value);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "SetAudioType Failed with error " + error);
+                    throw ExceptionFactory.CreateException(error);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Sets or Gets the recognition mode.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State must be Ready.
+        /// </pre>
+        public static RecognitionModeType RecognitionMode
+        {
+            get
+            {
+                RecognitionModeType mode;
+
+                ErrorCode error = VcMgrGetRecognitionMode(out mode);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "GetRecognitionMode Failed with error " + error);
+                    return RecognitionModeType.Undefined;
+                }
+
+                return mode;
+            }
+            set
+            {
+                ErrorCode error = VcMgrSetRecognitionMode(value);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "SetRecognitionMode Failed with error " + error);
+                    throw ExceptionFactory.CreateException(error);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Sets private data between app and engine.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="key">Private key</param>
+        /// <param name="data">Private data</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
```

Comment:
If this exception can be thrown in both cases, please write it as a single instance of `<exception cref="InvalidOperationException">`.